packages/handleRichTextBoxKeyEventNavigation/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "quip-apps-handle-richtextbox-key-event-navigation",
-    "version": "0.1.2-alpha.14",
+    "version": "0.2.0",
```

Comment:
Since we deploy with lerna, this change won't do anything (lerna will just overwrite it as part of CI). I haven't really come up with a good solution for deploying new versions outside of branch/bump/tag/publish - would you like me to publish this to NPM as 0.2.0?

Reply:
yes please!

Reply:
done!